sae_lens/sae_training_runner.py

```diff
@@ -62,12 +64,16 @@ def __init__(
             )
         else:
             self.model = override_model
+            self.model.to(self.cfg.device)
 
-        self.activations_store = ActivationsStore.from_config(
+        activations_store_cls = DRCActivationsStore if is_drc else ActivationsStore
+        self.activations_store = activations_store_cls.from_config(
             self.model,
             self.cfg,
             override_dataset=override_dataset,
         )
+        if is_drc:
+            self.cfg.training_tokens *= self.cfg.epochs
```

Comment:
Is this because, unlike normal SAE, we need to train for multiple epochs? Perhaps add a commenbt